vehicle/src/Vehicle/CommandLine.hs

```diff
@@ -454,6 +455,13 @@ outputAsJSONParser =
       <> help "Output the program as JSON instead of text."
       <> internal
 
+resourcesParser :: Parser Bool
+resourcesParser =
+  switch $
+    long "resources"
+      <> short 'r'
+      <> help "Output the resources outlined in a vehicle file. Only prints if typeCheck is successful."
```

Comment:
Maybe "After type-checking output the resources (e.g. networks, parameters, datasets) in the file as JSON"?